app/src/main/java/com/versobit/kmark/xhangouts/Setting.java

```diff
@@ -46,6 +46,18 @@
     UI_SEND_LOCK,
     UI_DISABLE_PROXIMITY,
     UI_APP_COLOR,
+    UI_INCOMING("incomingColor"),
+    UI_INCOMING_OTR("incomingColorOTR"),
+    UI_INCOMING_FONT("incomingFontColor"),
+    UI_INCOMING_FONT_OTR("incomingFontColorOTR"),
+    UI_INCOMING_LINK("incomingLinkColor"),
+    UI_INCOMING_LINK_OTR("incomingLinkColorOTR"),
+    UI_OUTGOING("outgoingColor"),
+    UI_OUTGOING_OTR("outgoingColorOTR"),
+    UI_OUTGOING_FONT("outgoingFontColor"),
+    UI_OUTGOING_FONT_OTR("outgoingFontColorOTR"),
+    UI_OUTGOING_LINK("outgoingLinkColor"),
+    UI_OUTGOING_LINK_OTR("outgoingLinkColorOTR"),
```

Comment:
I could do that. I didn't originally as I started from the XML and then wen't on to fix the orientation bug. I guess I got a little lazy and left custom names in place instead of changing things again.